practices/progressive-enhancement.md

```diff
@@ -28,6 +28,16 @@ Developing software using progressive enhancement also encourages Lean thinking,
 
 Finally by starting with just the most basic layer of content and moving on from there, progressive enhancement encourages a lighter, more performant delivery of software. For someone on a slow network, or with a device which can't support the layers of sophistication you've built into your site, they'll appreciate being able to access the content and basic functionality you've provided even while the bells and whistles are downloaded/parsed/executed (or none of the above).
 
+## Implementation
+
+We enable progressive enhancement at every level of our products.
+
+On the broadest level our implementation of [Graded Browser Support](graded-browser-support.md) lets us offer Core versions of our products that work for all users, in all situations, and which are then enhanced when the browser is deemed advanced enough.
+
+On the CSS level we build basic implementations first, using primitive CSS, and then enhance in Advanced browsers using the [cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade), [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), source order, and [feature queries](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports).
```

Comment:
it seems a bit odd to me to be calling out those features of CSS here, what are you driving at?

Reply:
This was an attempt to emphaise that PE takes place at both a high level (via CTM) and then on a low-level, per-component, per-file, and, well, per-everything basis. 

Was a big sleep fatigued when I wrote it, so someone pls shout if you think it's worth pursuing, or should be ditched for this PR.

Reply:
Re-reading I think I get you now (especially in conjunction with the following paragraph) 👍